atom/app/atom_content_client.cc

```diff
@@ -8,9 +8,46 @@
 #include <vector>
 
 #include "atom/common/chrome_version.h"
+#include "atom/common/options_switches.h"
+#include "base/command_line.h"
+#include "base/strings/string_split.h"
+#include "base/strings/string_util.h"
+#include "content/public/common/content_constants.h"
+#include "content/public/common/pepper_plugin_info.h"
+#include "ppapi/shared_impl/ppapi_permissions.h"
 
 namespace atom {
 
+int32 kPepperFlashPermissions = ppapi::PERMISSION_ALL_BITS;
```

Comment:
I think we can remove this definition and use `ppapi::PERMISSION_ALL_BITS` directly.